pipeline/tests/powdr_std.rs

```diff
@@ -44,7 +44,8 @@ fn split_gl_test() {
 fn arith_test() {
     let f = "std/arith_test.asm";
     verify_test_file::<GoldilocksField>(f, Default::default(), vec![]);
-    gen_estark_proof(f, Default::default());
+    // Otherwise, we get: thread 'arith_test' has overflowed its stack
+    // gen_estark_proof(f, Default::default());
```

Comment:
😢 I wonder why this happens. This line works just fine:
`cargo run --features halo2 -r pil test_data/std/arith_test.asm -o output -f --prove-with estark `

Also, I didn't add any columns or constraints in this PR (except maybe the main machine in `arith_test.asm`), so I don't see how it would make a difference for eStark?

Reply:
Maybe we should investigate this a bit. Is it only a problem for us?

Reply:
Looks like this problem just disappeared.

Reply:
Tests run in threads with limited memory iirc